programming/power.md

```diff
@@ -7,121 +9,127 @@ Power Board API
+- A fully charged battery will measure 12.6V.
+- The power board will turn off and signal a low battery at 10.2V.
+- The discharge curve is roughly linear between 11.4V and 10.4V.
 
 
 [Beeping](#beeping) {#beeping}
--------
+------------------------------
 
 The power board has a piezo buzzer which can beep.
 
-The `buzz` function accepts multiple parameters, depending on what you
-want to play. The first argument is the duration of the beep, in
-seconds. The later arguments are either the note you want to play, or
-the frequency of the buzzer (in Hertz). You have to specify which of
-note or frequency you're passing using a keyword argument, your code
-will fail otherwise.
+The `buzz` function accepts multiple parameters, depending on what you want to play.
+- The first argument is either the note you want to play, or the frequency of the buzzer in Hertz.
+- The second argument is the duration of the beep, in seconds.
 
-Theoretically, the piezo buzzer will buzz at any provided frequency, however
-humans can only hear between [20Hz and 20000Hz][pitch-range].
+The `Note` enum provides notes in [scientific pitch notation](https://en.wikipedia.org/wiki/Scientific_pitch_notation) between `C6` and `C8`.
+You can play other tones by providing a frequency.
 
-The `Note` enum provides notes in [scientific pitch notation][pitch-notation]
-between `C6` and `C8`. You can play other tones by providing a frequency.
+The frequency on the buzzer is limited from 8Hz to 10,000Hz
 
 <div class="info" markdown="1">
-  Calling `buzz` is non-blocking, which means it doesn't actually wait
-  for the piezo to stop buzzing before continuing with your code. If you
-  want to wait for the buzzing to stop, use the <code>blocking</code> argument!
+  Calling `buzz` is non-blocking, which means it doesn't actually wait for the piezo to stop buzzing before continuing with your code.
+  If you want to wait for the buzzing to stop, use the `blocking` argument!
 </div>
 
 ~~~~~ python
 from sr.robot3 import Note
 
 # Beep for 0.5s in D.
-R.power_board.piezo.buzz(0.5, Note.D6)
+R.power_board.piezo.buzz(Note.D6, 0.5)
 
 # Beep for 2s at 400Hz
-R.power_board.piezo.buzz(2, 400)
+R.power_board.piezo.buzz(400, 2)
 
 # Beep for 3s at 250Hz and wait for it to finish
-R.power_board.piezo.buzz(3, 250, blocking=True)
+R.power_board.piezo.buzz(250, 3, blocking=True)
 ~~~~~
 
-`ValueError` is raised if the note is not recognised or the frequency is not an integer.
 
+[Start Button](#start_button) {#start_button}
+---------------------------------------------
+
+You can manually wait for the start button to be pressed, not only at the start.
+
+~~~~~ python
+robot.wait_start()
+~~~~~
 
-[pitch-range]: https://en.wikipedia.org/wiki/Hearing_range#Humans
-[pitch-notation]: https://en.wikipedia.org/wiki/Scientific_pitch_notation
+This may be useful for testing, but be sure to remove it in the competition,
+as you won't be allowed to touch the start button after a match has begun!
```

Comment:
Nit: Note that it pauses execution until the button is pressed:

```suggestion
This method will block until the start button is pressed. This may be useful for testing, but be sure to remove it in the competition,
as you won't be allowed to touch the start button after a match has begun!
```

Reply:
b3e9be8